playbooks/common/openshift-cluster/upgrades/etcd/backup.yml

```diff
@@ -13,6 +12,8 @@
       role: etcd
       local_facts: {}
     when: "'etcd' not in openshift"
+  - set_fact:
+      timestamp: "{{ lookup('pipe', 'date +%Y%m%d%H%M%S') }}"
```

Comment:
I did this because without this it was actually re-evaluating the lookup each time `{{ timestamp }}` was used below which meant that the backup location it told you existed usually didn't unless the set of tasks happened within the same millisecond, on my test vm at least that never happened but maybe that's only because i've added additional backup steps.